### PR TITLE
Add MP 7.0 Plan Review

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

**Comment by emecas** [comment]
@Emily-Jiang  is this PR still a draft, right? or an early review is possibly before ballot is concluded?

**Comment by Emily-Jiang** [comment]
> @Emily-Jiang is this PR still a draft, right? or an early review is possibly before ballot is concluded?

Early review prior to the ballot.

File: microprofile-7.0/README.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+MicroProfile follows a train model for its Platform releases. Typically, three Platform releases are planned for each year in February, June, and October. MicroProfile releases components that are ready and have been approved for inclusion will be part of the Platform release. From this release onwards, Jakarta EE Core Profile will no longer be packaged in the MicroProfile BOM and the dependency has been made to be loosely coupled. MicroProfile Metrics specification is no longer included in the platform release, because the Metrics from MicroProfile Telemetry should be used instead. The specifications in MicroProfile 7.0 include:
+
+* Config 3.1
+* Fault Tolerance 5.0*
+* Health 4.0
+* JWT RBAC 3.0*
+* OpenAPI 4.0*
+* Rest Client 4.0*
+* Telemetry 2.0*
+
```
Comment (jclingan) [inline microprofile-7.0/README.adoc:16]: To be consistent with other content, we should add "Jakarta EE 10 Core Profile". Especially when everyone is asking if MicroProfile 7.0 aligns with Jakarta EE 10 or 11.

**Review by jclingan — APPROVED** [review]
Looks good

**Review by emecas — APPROVED** [review]
LGTM

File: microprofile-7.0/README.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+date: 2024-06-19
+summary: "MicroProfile 7.0 Release Plan"
+---
+
+MicroProfile follows a train model for its Platform releases. Typically, three Platform releases are planned for each year in February, June, and October. MicroProfile releases components that are ready and have been approved for inclusion will be part of the Platform release. From this release onwards, Jakarta EE Core Profile will no longer be packaged in the MicroProfile BOM and the dependency has been made to be loosely coupled. MicroProfile Metrics specification is no longer included in the platform release, because the Metrics from MicroProfile Telemetry should be used instead. The specifications in MicroProfile 7.0 include:
+
+* Config 3.1
+* Fault Tolerance 5.0*
+* Health 4.0
+* JWT RBAC 3.0*
```
Comment (Emily-Jiang) [inline microprofile-7.0/README.adoc:12]:
```suggestion
```

File: microprofile-7.0/README.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+date: 2024-06-19
+summary: "MicroProfile 7.0 Release Plan"
+---
+
+MicroProfile follows a train model for its Platform releases. Typically, three Platform releases are planned for each year in February, June, and October. MicroProfile releases components that are ready and have been approved for inclusion will be part of the Platform release. From this release onwards, Jakarta EE Core Profile will no longer be packaged in the MicroProfile BOM and the dependency has been made to be loosely coupled. MicroProfile Metrics specification is no longer included in the platform release, because the Metrics from MicroProfile Telemetry should be used instead. The specifications in MicroProfile 7.0 include:
+
+* Config 3.1
+* Fault Tolerance 5.0*
+* Health 4.0
+* JWT RBAC 2.1*
```
Comment (Emily-Jiang) [inline microprofile-7.0/README.adoc:12]:
```suggestion
* JWT RBAC 2.1
```

File: microprofile-7.0/README.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: "MicroProfile 7.0"
+date: 2024-06-19
+summary: "MicroProfile 7.0 Release Plan"
+---
+
+MicroProfile follows a train model for its Platform releases. Typically, three Platform releases are planned for each year in February, June, and October. MicroProfile releases components that are ready and have been approved for inclusion will be part of the Platform release. From this release onwards, Jakarta EE Core Profile will no longer be packaged in the MicroProfile BOM and the dependency has been made to be loosely coupled. MicroProfile Metrics specification is no longer included in the platform release, because the Metrics from MicroProfile Telemetry should be used instead. The specifications in MicroProfile 7.0 include:
```
Comment (emecas) [inline microprofile-7.0/README.adoc:7]: MicroProfile follows a regular release cadence with three updates planned annually in February, June, and October. Each release incorporates components that have completed development and gained community approval.

Key Changes in MicroProfile 7.0

 - Jakarta EE Decoupling: Starting with version 7.0, the Jakarta EE Core Profile is no longer automatically included in the MicroProfile Bill of Materials (BOM). Dependencies are now loosely coupled.

 - MicroProfile Metrics Deprecation: The dedicated MicroProfile Metrics specification has been removed. To gather application metrics, developers should leverage the Metrics functionality within MicroProfile Telemetry.

MicroProfile 7.0 Specifications

This release includes the following specifications:

Comment (jclingan) [reply microprofile-7.0/README.adoc:7]: I would rephrase the 2nd bullet from:

_MicroProfile Metrics Deprecation: The dedicated MicroProfile Metrics specification has been removed. To gather application metrics, developers should leverage the Metrics functionality within MicroProfile Telemetry._

to:

_MicroProfile Metrics as a Standalone Specification: The MicroProfile Metrics specification is being moved from the MicroProfile 7.0 Platform Specification to evolve as a Standalone Specification. MicroProfile Telemetry 2.0, which is already in the MicroProfile Platform Specification, has added Metrics functionality._

**Review by emecas — APPROVED** [review]
It reads better now, thanks to John and Emily's great work incorporating the changes!

**Comment by edbratt** [comment]
The plan should be updated to reflect Fault Tolerance is downgraded to 4.1 (from original 5.0)
